kubernetes/samples/charts/ingress-per-domain/README.md

```diff
@@ -0,0 +1,67 @@
+# Ingress per Domain Chart
+This chart is to deploy an Ingress resource in front of a WebLogic domain cluster. We support tow Ingress types: Traeafik and Voyager.
+
+## Prerequisites
+- The corresponding Ingress controller: Traefik or Voyager, is installed in the Kubernetes.
+- A WebLogic domain cluster is running in the Kubernetes.
+
+## Installing the Chart
+
+To install the chart with the release name my-ingress with given values.yaml:
+```
+# Change directory to the cloned git weblogic-kubernetes-operator repo.
+$ cd kubernetes/samples/charts
+$ helm install ingress-per-domain --name my-ingress --value values.yaml
+```
+The Ingress resource will be created to the same namespace as the WebLogic domain cluster.
```

Comment:
The Ingress resource will be created in the same namespace as the WebLogic domain cluster.